src/components/PricingTable/Unsubscribe.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Button, Modal } from 'antd';
+import get from 'lodash/get';
+import * as typeformEmbed from '@typeform/embed';
+import { deleteAppSubscription, getAppPlan } from '../../batteries/modules/actions';
+import { displayErrors } from '../../utils/helper';
+
+const TYPE_FORM_UNLOADED = 0;
+const TYPE_FORM_LOADED = 1;
+const TYPE_FORM_SUBMITTED = 2;
+
+class Unsubscribe extends Component {
+	constructor(props) {
+		super(props);
+		this.myRef = React.createRef();
+		this.state = { typeFormStep: TYPE_FORM_UNLOADED };
+	}
+
+	componentDidUpdate(prevProps) {
+		const { errors } = this.props;
+		displayErrors(errors, prevProps.errors, true);
+	}
+
+	embedTypescriptWidget = () => {
+		this.setState({ typeFormStep: TYPE_FORM_LOADED });
+		if (this.myRef.current) {
+			typeformEmbed.makeWidget(
+				this.myRef.current,
+				'https://siddharth31.typeform.com/to/QEktta',
+				{
+					hideFooter: true,
+					hideHeaders: true,
+					opacity: 0,
+					onSubmit: () => {
+						this.setState({ typeFormStep: TYPE_FORM_SUBMITTED });
+					},
+				},
+			);
+		}
+	};
+
+	deleteSubscription = () => {
+		const { deleteSubscription, fetchAppPlan } = this.props;
+		deleteSubscription().then((action) => {
+			if (get(action, 'payload')) {
+				fetchAppPlan();
+				this.cancelConfirmBox();
+			}
+		});
+	};
+
+	render() {
+		const { loading, onCancel } = this.props;
+		const { typeFormStep } = this.state;
+		return (
+			<Modal
+				visible
+				title="Unsubscribe from the current plan"
+				onCancel={onCancel}
+				footer={[
+					<Button key="back" onClick={onCancel}>
+						{/* eslint-disable-next-line react/no-unescaped-entities */}
+						I've changed my mind
+					</Button>,
```

Comment:
An extra comma is there is it intentional?

Reply:
if not there, throws eslint `comma-dangle`